core/src/main/java/org/infinispan/commands/functional/ReadWriteKeyCommand.java

```diff
@@ -130,4 +128,26 @@ public void init(ComponentRegistry componentRegistry) {
          ((InjectableComponent) f).inject(componentRegistry);
       }
    }
+
+   @Override
+   public final boolean isReturnValueExpected() {
+      return true;
+   }
+
+   @Override
+   public Metadata getMetadata() {
```

Comment:
@rvansa for unsupported operation exception on RWKVC I can create an issue to add support on ST for functional commands separetly as well